glossary.yml

```diff
@@ -588,13 +619,26 @@
     def: >
       FIXME
 
+- slug: "false"
+  en: 
+    term: "false"
+    def: > 
+       The logical (Boolean) state opposite of "true". Used in logic and programming to represent binary state of something. 
+       Often fluidly represented as true/false, on/off, 1/0. 
```

Comment:
```suggestion
```

False is rarely represented as "true" unless it's coming from a politician or [a malicious R package](https://github.com/Ironholds/jammr)